Cmake seems to write into wrong place `Build files have been written to: C:/Users/runneradmin/.cargo/registry/src/index.crates.io-6f17d22bba15001f/libcec-sys-4.0.2`?

```
2023-07-05T18:30:05.7473936Z [libcec-sys 4.0.2] CMake Deprecation Warning at CMakeLists.txt:3 (cmake_minimum_required):
2023-07-05T18:30:05.7474809Z [libcec-sys 4.0.2]   Compatibility with CMake < 2.8.12 will be removed from a future version of
2023-07-05T18:30:05.7475276Z [libcec-sys 4.0.2]   CMake.
2023-07-05T18:30:05.7475582Z [libcec-sys 4.0.2] 
2023-07-05T18:30:05.7476070Z [libcec-sys 4.0.2]   Update the VERSION argument <min> value or use a ...<max> suffix to tell
2023-07-05T18:30:05.7477678Z [libcec-sys 4.0.2]   CMake that the project does not need compatibility with older versions.
2023-07-05T18:30:05.7478163Z [libcec-sys 4.0.2] 
2023-07-05T18:30:05.7478454Z [libcec-sys 4.0.2] 
2023-07-05T18:30:05.7782003Z [libcec-sys 4.0.2] -- Performing Test CMAKE_HAVE_LIBC_PTHREAD
2023-07-05T18:30:06.0051512Z [libcec-sys 4.0.2] -- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
2023-07-05T18:30:06.0052245Z [libcec-sys 4.0.2] -- Looking for pthread_create in pthreads
2023-07-05T18:30:06.3224526Z [libcec-sys 4.0.2] -- Looking for pthread_create in pthreads - not found
2023-07-05T18:30:06.3225466Z [libcec-sys 4.0.2] -- Looking for pthread_create in pthread
2023-07-05T18:30:06.6108940Z [libcec-sys 4.0.2] -- Looking for pthread_create in pthread - not found
2023-07-05T18:30:06.6265337Z [libcec-sys 4.0.2] -- Found Threads: TRUE  
2023-07-05T18:30:06.6289164Z [libcec-sys 4.0.2] -- Looking for __atomic_fetch_add_4 in atomic
2023-07-05T18:30:06.9086406Z [libcec-sys 4.0.2] -- Looking for __atomic_fetch_add_4 in atomic - not found
2023-07-05T18:30:06.9087358Z [libcec-sys 4.0.2] -- Performing Test HAS_BUILTIN_SYNC_ADD_AND_FETCH
2023-07-05T18:30:07.1441073Z [libcec-sys 4.0.2] -- Performing Test HAS_BUILTIN_SYNC_ADD_AND_FETCH - Failed
2023-07-05T18:30:07.1441974Z [libcec-sys 4.0.2] -- Performing Test HAS_BUILTIN_SYNC_SUB_AND_FETCH
2023-07-05T18:30:07.3543868Z [libcec-sys 4.0.2] -- Performing Test HAS_BUILTIN_SYNC_SUB_AND_FETCH - Failed
2023-07-05T18:30:07.3544920Z [libcec-sys 4.0.2] -- Performing Test HAS_BUILTIN_SYNC_VAL_COMPARE_AND_SWAP
2023-07-05T18:30:07.5632021Z [libcec-sys 4.0.2] -- Performing Test HAS_BUILTIN_SYNC_VAL_COMPARE_AND_SWAP - Failed
2023-07-05T18:30:07.5707441Z [libcec-sys 4.0.2] -- Configuring done (13.8s)
2023-07-05T18:30:07.5980431Z [libcec-sys 4.0.2] -- Generating done (0.0s)
2023-07-05T18:30:07.6053614Z [libcec-sys 4.0.2] -- Build files have been written to: C:/Users/runneradmin/.cargo/registry/src/index.crates.io-6f17d22bba15001f/libcec-sys-4.0.2
2023-07-05T18:30:07.6339753Z [libcec-sys 4.0.2] The system cannot find the path specified.
```


What does `2023-07-05T18:30:15.1870427Z [libcec-sys 4.0.2] The system cannot find the path specified.` mean?

Anyways, clearly issues with cmake again, check last error
```
2023-07-05T18:30:15.1870427Z [libcec-sys 4.0.2] The system cannot find the path specified.
2023-07-05T18:30:15.1870913Z [libcec-sys 4.0.2]  --------------------------------------
2023-07-05T18:30:15.1871346Z [libcec-sys 4.0.2]  Generating cmake project:
2023-07-05T18:30:15.1872112Z [libcec-sys 4.0.2]  Architecture = amd64
2023-07-05T18:30:15.1872560Z [libcec-sys 4.0.2]  Project type = "NMake Makefiles"
2023-07-05T18:30:15.1872992Z [libcec-sys 4.0.2]  Cmake ARCH   = 
2023-07-05T18:30:15.1873697Z [libcec-sys 4.0.2]  Project      = "D:\a\cec-rs\cec-rs\target\x86_64-pc-windows-msvc\debug\build\libcec-sys-bde96935d55bde00\out\vendor\windows\.."
2023-07-05T18:30:15.1874641Z [libcec-sys 4.0.2]  Target       = "D:\a\cec-rs\cec-rs\target\x86_64-pc-windows-msvc\debug\build\libcec-sys-bde96935d55bde00\out\libcec_build\cmake\amd64"
2023-07-05T18:30:15.1875560Z [libcec-sys 4.0.2]  Install      = "D:\a\cec-rs\cec-rs\target\x86_64-pc-windows-msvc\debug\build\libcec-sys-bde96935d55bde00\out\libcec_build\amd64"
2023-07-05T18:30:15.1876144Z [libcec-sys 4.0.2]  Build type   = Debug
2023-07-05T18:30:15.1876560Z [libcec-sys 4.0.2]  Toolchain    = Visual Studio 16 2019
2023-07-05T18:30:15.1877015Z [libcec-sys 4.0.2]  --------------------------------------
2023-07-05T18:30:15.1877383Z [libcec-sys 4.0.2] 
2023-07-05T18:30:15.1878603Z [libcec-sys 4.0.2] CMake Error: The source "D:/a/cec-rs/cec-rs/target/x86_64-pc-windows-msvc/debug/build/libcec-sys-bde96935d55bde00/out/vendor/CMakeLists.txt" does not match the source "D:/a/cec-rs/cec-rs/target/x86_64-pc-windows-msvc/debug/build/libcec-sys-bde96935d55bde00/out/vendor/src/platform/CMakeLists.txt" used to generate cache.  Re-run cmake with a different source directory.
```

Understandably `cec.lib` is not created. This is probably a symptom, not the cause.
```
2023-07-05T18:30:16.7364465Z   = note: LINK : fatal error LNK1181: cannot open input file 'cec.lib'
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -99,7 +99,7 @@ jobs:
                   profile: minimal
                   target: ${{ matrix.job.target }}
                   override: true
-            - uses: Swatinem/rust-cache@v1
+            - uses: Swatinem/rust-cache@v2
             # libcec and p8-platform from apt repo
             - if: ${{ !startsWith(matrix.job.libcec, 'vendored') && runner.os == 'Linux' }}
               name: Install libcec(-dev) and build dependencies

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -24,68 +24,68 @@ jobs:
                       use-cross: false
                       libcec: vendored-libcec
                       expected_libcec_abi: 6
-                    - os: ubuntu-latest
-                      target: aarch64-unknown-linux-gnu
-                      use-cross: true
-                      libcec: vendored-libcec
-                      expected_libcec_abi: 6
-                    - os: ubuntu-latest
-                      target: i686-unknown-linux-gnu
-                      use-cross: true
-                      libcec: vendored-libcec
-                      expected_libcec_abi: 6
-                    - os: ubuntu-latest
-                      target: arm-unknown-linux-gnueabi
-                      use-cross: true
-                      libcec: vendored-libcec
-                      expected_libcec_abi: 6
-                    - os: ubuntu-latest
-                      target: armv7-unknown-linux-gnueabihf
-                      use-cross: true
-                      libcec: vendored-libcec
-                      expected_libcec_abi: 6
-                    - os: ubuntu-latest
-                      target: mips-unknown-linux-gnu
-                      use-cross: true
-                      libcec: vendored-libcec
-                      expected_libcec_abi: 6
-                    - os: ubuntu-latest
-                      target: mips64-unknown-linux-gnuabi64
-                      use-cross: true
-                      libcec: vendored-libcec
-                      expected_libcec_abi: 6
-                    - os: ubuntu-latest
-                      target: mips64el-unknown-linux-gnuabi64
-                      use-cross: true
-                      libcec: vendored-libcec
-                      expected_libcec_abi: 6
-                    - os: ubuntu-latest
-                      target: mipsel-unknown-linux-gnu
-                      use-cross: true
-                      libcec: vendored-libcec
-                      expected_libcec_abi: 6
-                    - os: ubuntu-latest
-                      target: x86_64-unknown-linux-gnu
-                      use-cross: true
-                      libcec: vendored-libcec
-                      expected_libcec_abi: 6
-                    #
-                    # libcec discovery with pkg config
-                    #
-                    - os: ubuntu-latest
-                      target: x86_64-unknown-linux-gnu
-                      use-cross: false # Important that we do not run cross so package install shows up
-                      libcec: "libcec4"
-                      libcec-dev: "libcec-dev=4*"
-                      pkg-config: true
-                      expected_libcec_abi: 4
-                    - os: ubuntu-latest
-                      target: x86_64-unknown-linux-gnu
-                      use-cross: false # Important that we do not run cross so package install shows up
-                      libcec: "libcec6"
-                      libcec-dev: "libcec-dev=6*"
-                      pkg-config: true
-                      expected_libcec_abi: 6
+                    # - os: ubuntu-latest
+                    #   target: aarch64-unknown-linux-gnu
+                    #   use-cross: true
+                    #   libcec: vendored-libcec
+                    #   expected_libcec_abi: 6
+                    # - os: ubuntu-latest
+                    #   target: i686-unknown-linux-gnu
+                    #   use-cross: true
+                    #   libcec: vendored-libcec
+                    #   expected_libcec_abi: 6
+                    # - os: ubuntu-latest
+                    #   target: arm-unknown-linux-gnueabi
+                    #   use-cross: true
+                    #   libcec: vendored-libcec
+                    #   expected_libcec_abi: 6
+                    # - os: ubuntu-latest
+                    #   target: armv7-unknown-linux-gnueabihf
+                    #   use-cross: true
+                    #   libcec: vendored-libcec
+                    #   expected_libcec_abi: 6
+                    # - os: ubuntu-latest
+                    #   target: mips-unknown-linux-gnu
+                    #   use-cross: true
+                    #   libcec: vendored-libcec
+                    #   expected_libcec_abi: 6
+                    # - os: ubuntu-latest
+                    #   target: mips64-unknown-linux-gnuabi64
+                    #   use-cross: true
+                    #   libcec: vendored-libcec
+                    #   expected_libcec_abi: 6
+                    # - os: ubuntu-latest
+                    #   target: mips64el-unknown-linux-gnuabi64
+                    #   use-cross: true
+                    #   libcec: vendored-libcec
+                    #   expected_libcec_abi: 6
+                    # - os: ubuntu-latest
+                    #   target: mipsel-unknown-linux-gnu
+                    #   use-cross: true
+                    #   libcec: vendored-libcec
+                    #   expected_libcec_abi: 6
+                    # - os: ubuntu-latest
+                    #   target: x86_64-unknown-linux-gnu
+                    #   use-cross: true
+                    #   libcec: vendored-libcec
+                    #   expected_libcec_abi: 6
+                    # #
+                    # # libcec discovery with pkg config
+                    # #
+                    # - os: ubuntu-latest
+                    #   target: x86_64-unknown-linux-gnu
+                    #   use-cross: false # Important that we do not run cross so package install shows up
+                    #   libcec: "libcec4"
+                    #   libcec-dev: "libcec-dev=4*"
+                    #   pkg-config: true
+                    #   expected_libcec_abi: 4
+                    # - os: ubuntu-latest
+                    #   target: x86_64-unknown-linux-gnu
+                    #   use-cross: false # Important that we do not run cross so package install shows up
+                    #   libcec: "libcec6"
+                    #   libcec-dev: "libcec-dev=6*"
+                    #   pkg-config: true
+                    #   expected_libcec_abi: 6
         steps:
             - name: Checkout repository
               uses: actions/checkout@v2

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -23,7 +23,7 @@ derive_builder = '0.10.2'
 num-traits = '0.2.14'
 
 [dependencies.libcec-sys]
-git = "https://github.com/ssalonen/libcecy-sys.git"
+git = "https://github.com/ssalonen/libcec-sys.git"
 branch = "ci-win-debug"
 
 

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -23,7 +23,9 @@ derive_builder = '0.10.2'
 num-traits = '0.2.14'
 
 [dependencies.libcec-sys]
-version = '4.0.0'
+git = "https://github.com/ssalonen/libcecy-sys.git"
+branch = "ci-win-debug"
+
 
 [dependencies.enum-repr-derive]
 version = '0.2.0'

